docs/cmd-line-flags.md

```diff
@@ -15,9 +15,8 @@ does not need to be set.
 
 **`-instance (string)`**
 
-The instance to target when calling the remote execution service via gRPC. For
-Remote Build Execution, the format is
-`projects/_project-id_/instances/_instance-name_`.
```

Comment:
Looking at this doc line again, I think what reclient is mentioning here is accurate. Instance names are user specified and don't always have to be `default_instance`. So I would rather we change the documentation of remote-apis-sdks to reflect that.

If its ok with you, can you do an inverse of this patch and send a PR to remote-apis-sdks?